app/_data/docs_nav_konnect.yml

```diff
@@ -102,10 +102,6 @@
           url: /dev-portal/administrators/auto-approve-devs-apps
         - text: Public Portal
           url: /dev-portal/administrators/public-portal
-        - text: Customization
-          items:
-          - text: Customizing the Dev Portal Appearance
```

Comment:
Since you just deleted a page, you'll need to set up a redirect for it. Even if we don't have links in the docs to this page, there might still be something external.

In `app/_redirects`, add a line with two URLs: the relative URL you're redirecting from (`/konnect/dev-portal/administrators/customization/themes`) to the target page (`/konnect/dev-portal`). It'll just look like this:

```
/konnect/dev-portal/administrators/customization/themes         /konnect/dev-portal
```